Modules/DisableDevice.cs

```diff
@@ -176,6 +186,12 @@ public static void UpdateDisableDevices()
                 if (Options.DisableAirshipVital.GetBool())
                     consoles.DoIf(x => x.name == "panel_vitals", x => x.gameObject.GetComponent<CircleCollider2D>().enabled = false || ignore);
                 break;
+            case 5:
+                if (Options.DisableFungleCamera.GetBool())
+                    consoles.DoIf(x => x.name == "BinocularsSecurityConsole", x => x.gameObject.GetComponent<PolygonCollider2D>().enabled = false || ignore);
+                if (Options.DisableFungleCamera.GetBool())
+                    consoles.DoIf(x => x.name == "VitalsConsole", x => x.gameObject.GetComponent<BoxCollider2D>().enabled = false || ignore);
+                break;
```

Comment:
Looks like there is an error here, you need to use `if (Options.DisableFungleVital.GetBool())`